service/controller/v16/resource/deployment/worker_deployment.go

```diff
@@ -55,6 +55,13 @@ func newWorkerDeployments(customObject v1alpha1.KVMConfig) ([]*extensionsv1.Depl
 				},
 			},
 			Spec: extensionsv1.DeploymentSpec{
+				Selector: apismetav1.LabelSelector{
+					MatchLabels: map[string]string{
+						key.LabelApp: key.WorkerID,
+						"cluster":    key.ClusterID(customObject),
+						"node":       workerNode.ID,
```

Comment:
Do we need to use "old-style" labels here?

Reply:
That is the thing. We have to for the migration and to just make it work for now. When all pods have the new and the old labels, we can switch and remove the old ones completely.